### PR TITLE
Add some window factories

### PR DESCRIPTION
**Pull request by nyan-ko**
Title. not every window has a factory right now, will be made later

### CODE REVIEW

File: src/main/java/ui/factories/GameWindowFactory.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import ui.windows.GameDrawDelegator;
+import ui.windows.GamePlayDelegator;
+import ui.windows.GameSkipDelegator;
+import ui.windows.GameWindow;
+import ui.windows.Window;
+
+public class GameWindowFactory implements WindowFactory {
+
+    @Override
+    public Window createWindow() {
```
Comment (SMateen21) [inline src/main/java/ui/factories/GameWindowFactory.java:16]: Id only add some light Javadocs for this and the other two methods here.

**Review by SMateen21 — COMMENTED** [review]
Besides documentation, everything looks good to me.

**Review by lilyxmeng — APPROVED** [review]
Add Javadocs as Solemann said otherwise good to merge.

**Review by meiiph — APPROVED** [review]
Javadocs added.

**Review by SMateen21 — APPROVED** [review]
Looks good to me.

**Review by lilyxmeng — APPROVED** [review]

**Pull request merged**